docs/zh/get-started/Validator-Node.md

```diff
@@ -97,25 +99,17 @@ iriscli stake validator  {address-validator-operator}  --chain-id={chain-id} --n
 iriscli status --node=tcp://localhost:26657 
 ```
 
-你应该可以看到节点的`power`字段返回值大于0。
+你应该可以看到节点的`voting_power`字段返回值大于0。
 
 ### 编辑验证人信息
 
 你可以通过以下命令修改验证人的描述信息，验证人的名称默认为`--moniker`字段。
 你应该在`details`字段注明自定义的信息。
 
 ```
-iriscli stake edit-validator --from= {val-name}  --moniker="choose a moniker"  --website="https://irisnet.org"  --details="team" --chain-id=irishub-8000 
-   --node=tcp://localhost:26657 --fee=0.3iris  
-```
-### 查询验证人信息
-
-你可以通过以下命令查询验证人的信息：
-
-```
-iriscli stake validator {address-validator-operator} --chain-id=<chain-id> 
+iriscli stake edit-validator --from=<key name> --moniker=<your_custom_name> --website=<your_website> --details=<your_details> --chain-id=<chain-id> --node=tcp://localhost:26657 --fee=0.3iris  
```

Comment:
Maybe we should add some descriptions about `identity` when creating and editing validators.

Reply:
fixed